packages/lab/src/typography/Text.css

```diff
@@ -26,17 +22,12 @@
   word-break: break-word;
 }
 
-.uitkText-lazy {
-  transition: opacity 0.1s ease-in-out;
-  opacity: 0;
-}
-
 /* Heading 1 */
 
 h1.uitkText,
 .uitkText-h1.uitkText {
   font-size: var(--uitk-text-h1-fontSize);
-  line-height: var(--uitk-text-h1-lineHeight);
+  line-height: var(--uitk-text-lineHeight-backwardsCompat, var(--uitk-text-h1-lineHeight));
```

Comment:
we should have --uitkText-lineHeight as the first var and then set that in a .backwardsCompat.uitkText class